docs/xamarin-forms/user-interface/text/entry.md

```diff
@@ -67,9 +67,54 @@ The options for the keyboard type are:
 - **Telephone** &ndash; used when entering telephone numbers
 - **Url** &ndash; used for entering file paths and web addresses
 
+The following will create an Entry with keyboard type [`Keyboard.Chat`](xref:Xamarin.Forms.Keyboard.Chat) in XAML:
+
+```xaml
+<Entry ... Keyboard="Chat" />
```

Comment:
IMO, the "..." are irrelevant here since we want to demonstrate keyboard function only.